main.go

```diff
@@ -81,7 +92,60 @@ func main() {
 	fmt.Println()
 	log.Infof("$ apt-get %s", command.PrintableCommandArgs(false, cmdArgs))
 	if err := command.RunCommand("apt-get", cmdArgs...); err != nil {
-		log.Errorf("Can't install packages:  %s", err)
-		os.Exit(1)
+		fail("Can't install packages:  %s", err)
+	}
+}
+
+func applyCacheConfig(cacheConfig string) error {
+	switch cacheConfig {
+	case cacheInputNone:
+		return applyNoCache()
+	case cacheInputAll:
+		return applyAllCache()
+	default:
+		return fmt.Errorf("invalid cache level, no such configuration: %s", cacheConfig)
+	}
+}
+
+func applyNoCache() error {
+	if err := disablePackageCache(); err != nil {
+		return fmt.Errorf("could not disable package cache: %s", err)
+	}
+	if err := removeDownloadedPackages(); err != nil {
+		return fmt.Errorf("could not remove downloaded package cache: %s", err)
+	}
+	if err := removeLookupFiles(); err != nil {
+		return fmt.Errorf("could not remove lookup files: %s", err)
 	}
+	return nil
+}
+
+func applyAllCache() error {
+	if err := removeDockerCleanFile(); err != nil {
+		return fmt.Errorf("could not remove docker clean file: %s", err)
+	}
+	c := cache.New()
+	c.IncludePath("/var/cache/apt/archives")
+	if err := c.Commit(); err != nil {
+		return fmt.Errorf("could not add packages to cache: %s", err)
+	}
+	return nil
+}
+
+func disablePackageCache() error {
+	cacheFilePth := "/etc/apt/apt.conf.d/00_disable-cache"
+	cacheFileCnt := `Dir::Cache::pkgcache "";` + "\n" + `Dir::Cache::srcpkgcache "";` + "\n" + `Dir::Cache "";` + "\n" + `Dir::Cache::archives "";`
+	return fileutil.AppendStringToFile(cacheFilePth, cacheFileCnt)
+}
+
+func removeDownloadedPackages() error {
+	return command.RemoveAll("/var/cache/apt/archives/")
+}
+
+func removeLookupFiles() error {
+	return command.RemoveAll("/var/cache/apt/pkgcache.bin", "/var/cache/apt/pkgcache.bin")
```

Comment:
`"/var/cache/apt/pkgcache.bin"` is passed twice. Bug or feature? :)

Reply:
Sorry, fixed it.